pkg/osquery/tables/macos_software_update/recommended_updates_table.go

```diff
@@ -0,0 +1,113 @@
+//go:build darwin
+// +build darwin
+
+package macos_software_update
+
+/*
+#cgo darwin CFLAGS: -DDARWIN -x objective-c
+#cgo darwin LDFLAGS: -framework Cocoa
+#import <Cocoa/Cocoa.h>
+#import <SUSharedPrefs.h>
+#include "sus.h"
+*/
+import "C"
+import (
+	"context"
+	"strconv"
+	"strings"
+
+	"github.com/go-kit/kit/log"
+	"github.com/go-kit/kit/log/level"
+	"github.com/kolide/launcher/pkg/dataflatten"
+	"github.com/kolide/launcher/pkg/osquery/tables/dataflattentable"
+	"github.com/kolide/launcher/pkg/osquery/tables/tablehelpers"
+	"github.com/osquery/osquery-go/plugin/table"
+)
+
+const softwareUpdateToolPath = "/usr/sbin/softwareupdate"
+const softwareUpdateListArg = "--list"
+const softwareUpdateNoScanArg = "--no-scan"
+
+var updatesData []map[string]interface{}
+var doNotScan bool
```

Comment:
I'd declare these in `generate`. Might be an extra allocation, but it feels cleaner

Reply:
I left `updatesData` there since I need to reference it in the callbacks from C.